Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://hosted.weblate.org/projects/jasp/jasp-desktop/) for JASP/JASP desktop.



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/jasp/-/jasp-desktop/horizontal-auto.svg)

Commit:
Translated using Weblate (Dutch)

Currently translated at 99.9% (4854 of 4857 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/nl/

Commit:
Translated using Weblate (Dutch)

Currently translated at 92.7% (1996 of 2153 strings)

Translation: JASP/JASP engine
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-engine/nl/

Commit:
Translated using Weblate (Dutch)

Currently translated at 99.9% (4854 of 4857 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/nl/

Commit:
Translated using Weblate (Dutch)

Currently translated at 96.5% (2078 of 2153 strings)

Translation: JASP/JASP engine
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-engine/nl/

Commit:
Translated using Weblate (Dutch)

Currently translated at 97.5% (2100 of 2153 strings)

Translation: JASP/JASP engine
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-engine/nl/

Commit:
Translated using Weblate (Dutch)

Currently translated at 99.9% (2152 of 2153 strings)

Translation: JASP/JASP engine
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-engine/nl/

Commit:
Translated using Weblate (Dutch)

Currently translated at 99.9% (4854 of 4857 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/nl/

JoKeyser committed:
Translated using Weblate (German)

Currently translated at 98.8% (4803 of 4857 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/de/

JoKeyser committed:
Translated using Weblate (German)

Currently translated at 81.7% (1760 of 2153 strings)

Translation: JASP/JASP engine
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-engine/de/

vandenman commented on JASP-Engine/JASP/po/R-nl.po:
```diff
@@ -132,149 +114,138 @@ msgid ""
-#, fuzzy
 msgid "Proportion of Successes"
-msgstr "Proportie"
+msgstr "Proportie van successen"
```
```suggestion
msgstr "Proportie successen"
```

FransMeerhoff replied:
Ok

vandenman commented on JASP-Engine/JASP/po/R-nl.po:
```diff
@@ -132,149 +114,138 @@ msgid ""
 "not defined."
-msgstr "U kan niet alle prior modelkansen gelijk aan nul zetten."
+msgstr ""
+"De grafiek kan niet worden gemaakt omdat de waarschijnlijkheden van het "
+"posterior model niet zijn gedefinieerd."
```
```suggestion
"De grafiek kan niet worden gemaakt omdat de posterior model kansen "
"niet zijn gedefinieerd."
```
I'm not sure why this is on two lines instead of one though.

FransMeerhoff replied:
Ok

vandenman commented on JASP-Engine/JASP/po/R-nl.po:
```diff
@@ -132,149 +114,138 @@ msgid ""
 
 msgid "Sequential Posterior Updating"
-msgstr ""
+msgstr "Sequentiële Posterior Bijweken"
```
```suggestion
msgstr "Sequentiële Posterior Bijwerken"
```
typo?

FransMeerhoff replied:
Ok

vandenman commented on JASP-Engine/JASP/po/R-nl.po:
```diff
@@ -571,6 +751,13 @@ msgid ""
+"de voorspellingen voor 'N' toekomstige waarnemingen ('Toekomstige "
+"waarnemingen') een binomiale verdeling met grootte N en toevalsparameter %2$"
+"s die gelijk is aan de locatie van de prior verdeling. %5$s Voor een model "
+"met een bètavorige verdeling voor parameter %1$s is de voorspellende "
```
```suggestion
"met een bèta prior verdeling voor parameter %1$s is de voorspellende "
```

vandenman commented on JASP-Engine/JASP/po/R-nl.po:
```diff
@@ -646,45 +874,78 @@ msgid ""
 
 msgid ""
 "The 'Conditional' option shows all predictive accuracies independently, as "
 "ifthey were considered as individual models (without the existence of other "
```
```suggestion
"if they were considered as individual models (without the existence of other "
```
probably not for this PR, but that looks like a typo in the English.

FransMeerhoff replied:
Ok

vandenman commented on JASP-Engine/JASP/po/R-nl.po:
```diff
@@ -3399,7 +3641,7 @@ msgid ""
 msgstr ""
 "Het model kon niet passend gemaakt worden. Controleer a.u.b. het volgende: "
-"Heeft u ten minste n=2 onderzoeken? Als de prior afgekapt is, is hij dan "
+"Heeft u ten minste n=2 onderzoeken? Als de prior is afgekapt , is hij dan "
```
```suggestion
"Heeft u ten minste n=2 onderzoeken? Als de prior is afgekapt, is hij dan "
```

FransMeerhoff replied:
Ok

vandenman reviewed (changes requested):
I only looked at the Dutch translations, some comments below.

Commit:
Translated using Weblate (Dutch)

Currently translated at 99.9% (2152 of 2153 strings)

Translation: JASP/JASP engine
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-engine/nl/
